templates/definition/charger/homematic.yaml

```diff
@@ -0,0 +1,65 @@
+template: homematic
+products:
+  - brand: Homematic eQ-3 AG (ELV)
+    description:
+      generic: Homematic IP + BidCoS
+group: switchsockets
+params:
+  - name: host
+  - name: devicetype
+    description:
+      de: Homematic Gerätetyp (IP or BidCoS)
+      en: Homematic device type (IP or BidCoS)
+    required: true
+    mask: false
+    default: "IP"
```

Comment:
Freitexteingabe für eine Entweder-Oder-Eingabe ist von der Usability eher blöd. Vor allem wenn wir das nachher in einem Webinterface nutzen wollen. Wir könnten das als `valuetype: bool` abbilden. Für einige Parameter gibts ja auch ne Enum Eingabe (Bsp.: `choice: ["grid", "pv"]`). Da weiß ich allerdings nicht ob man die für jeden Parameter aktivieren kann oder ob das explizit für `usage` und `modbus` hinterlegt ist.

Reply:
Hatte es mit choice versucht, ging aber nicht. Müsste schauen, wo ich `choice` für devicetype einbauen muss. Ich schau mal ...

Reply:
Ich mach es ganz anders: Da es ja bisher nur diesen einen Typ der Steckdose gibt, lass ich die Auswahl einfach weg. :-)
Let's keep it simple ...